packages/repository/src/relations/has-one/index.ts

```diff
@@ -5,3 +5,4 @@
 
 export * from './has-one.decorator';
 export * from './has-one-repository.factory';
+export * from './has-one.repository';
```

Comment:
Let's keep the order of exports consistent with other places please:
 1. decorator
 2. repository
 3. factory

https://github.com/strongloop/loopback-next/blob/fed9956d760426766190b9c53cf7aaeed12d29f1/packages/repository/src/relations/has-many/index.ts#L6-L8

Reply:
Ok